cli/src/main/java/org/jboss/as/cli/handlers/OperationRequestHandler.java

```diff
@@ -184,18 +184,13 @@ private ModelNode validateRequest(CommandContext ctx, ModelNode request) throws
                     throw new CommandFormatException("Operation '" + operationName + "' does not expect any property.");
                 }
             }
```

Comment:
Yeah, it's wrong in master. Sorry, I wasn't clear. I only mentioned it in case you felt like deleting it.